Tests/UnitTests/Scripts/tests.js

```diff
@@ -264,6 +264,60 @@ describe("clearTimeout", function () {
     });
 });
 
+describe("WebSocket", function () {
```

Comment:
Might be good to have some tests for the error event.

Reply:
Discussed offline, filed an issue for this